test/functional/test-platform.js

```diff
@@ -59,10 +63,29 @@ describe('Platform', () => {
     expect(platform.getIosMajorVersion()).to.equal(iosMajorVersion);
   }
 
+  function testStandalone(userAgentString, standAloneBoolean) {
+    const platform = new Platform({
+      navigator: {
+        standalone: standAloneBoolean,
+        userAgent: userAgentString,
+      },
+    });
+    expect(platform.isStandalone()).to.equal(isStandalone);
+  }
+
   it('should tolerate empty or null', () => {
     testUserAgent(null);
     testUserAgent('');
     testUserAgent(' ');
+    testStandalone(null, null);
+    testStandalone(null, '');
+    testStandalone(null, ' ');
+    testStandalone('', null);
+    testStandalone('', '');
+    testStandalone('', ' ');
+    testStandalone(' ', null);
+    testStandalone(' ', '');
+    testStandalone(' ', ' ');
```

Comment:
Nice, your thoroughness is awesome. But realistically the value of `navigator.standalone` will never be a string, so in this test you can just call `testStandalone` with `null` for the second argument.

It wouldn't cause any problems, but it might be a distraction for someone looking at this file later trying to understand how it works.